### ISSUE TITLE
Project remains empty after opening a file

### ISSUE THREAD

**Issue by vaertgeerts** [OpenShot/openshot-qt#770]
**System Details:**
* Operating System / Distro: Win 7
* OpenShot Version: 2.3.4
* _Please attach log files if crash_

**Issue Description and steps to reproduce:**
Hello,

I made a project in OpenShot Video editor 2.3.4, but now when I try to reopen the project file (*.osp), the program doesn't doe anything. It just remains empty.
I think all photo and video files I use, are still in the same subfolder.
Another project doesn't have this issue.

What can be wrong?
Vincent
[video 22 juni.zip](https://github.com/OpenShot/openshot-qt/files/1122422/video.22.juni.zip)

**Comment by jariahtiainen** [comment]
Same problem here. 
Win 7 64bit, OpenShot Version: 2.3.4

**Comment by DOSn3rd** [comment]
I just got the same type of problem.

Opening by doubleclick on file yielded nothing, as did using the open project menu entry.
This happened after OpenShot crashed during exporting to file. Before that happened the project file opened like it should. 
Uninstalling, removing all settings in the .openshot-qt folder and then reinstalling did not help.

What **did** help was opening the file in WordPad, copying all the contents to a new file (in my case test.osp instead of the original filename) and then opening that file instead, then it all worked flawlessly.

I have attached the log from when i doubleclicked the osp file and it didn't work.

[openshot-qt_20170709.txt](https://github.com/OpenShot/openshot-qt/files/1133985/openshot-qt_20170709.txt)
